srcpkgs/python3-pgzero/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'python3-pgzero'
 pkgname=python3-pgzero
-version=1.2
-revision=3
+version=1.2.1
+revision=1
 wrksrc="pgzero-${version}"
 build_style=python3-module
-pycompile_module="pgzero pgzrun.py"
 hostmakedepends="python3-setuptools"
-depends="python3-setuptools python3-pygame python3-numpy"
+depends="python3-setuptools python3-pygame>=2.0.0 python3-numpy"
 short_desc="Zero-boilerplate games programming framework based on Pygame (Python3)"
 maintainer="Peter Bui <pbui@github.bx612.space>"
 license="LGPL-3.0-or-later"
 homepage="https://pygame-zero.readthedocs.io/en/stable/"
 distfiles="${PYPI_SITE}/p/pgzero/pgzero-${version}.tar.gz"
-checksum=91e641d545c6235a24719dea0fd83c8429e92a8b5ab1756bef145128e8db9017
+checksum=8cadc020f028cbac3e0cbd3bb9311a1c045f1deedac7917ff433f986c38e6106
+make_check=no
```

Comment:
Do leave a comment as to why tests are disabled :)

Reply:
Done!